README.md

```diff
@@ -75,6 +75,17 @@ img_info = FormatParser.parse(File.open("myimage.jpg", "rb"))
 JSON.pretty_generate(img_info) #=> ...
 ```
 
+To convert the result to a Hash
```

Comment:
If our objective is to convert to a Hash, is the `to_h` method not a better / more intuitive fit?

Reply:
I agree on it is a bit confusing! 

Some classes uses to_h like [Array#to_h](https://apidock.com/ruby/v2_5_5/Array/to_h) but other use as_json like [ActiveModel::Serializers::JSON#as_json](https://apidock.com/rails/ActiveModel/Serializers/JSON/as_json)

In this case, the method `as_json` already exists because the classes include the module `FormatParser::AttributesJSON`, so I chose to not change the name.

wdyt?

Reply:
Fair point, maybe change the wording a bit to "To convert the result to a Hash or a structure suitable for JSON serialization" to justify the use of `as_json` a bit better?

Reply:
great, much better!